awx/main/tasks.py

```diff
@@ -3088,8 +3088,8 @@ def pod_definition(self):
             pod_spec_override = parse_yaml_or_json(self.task.instance.instance_group.pod_spec_override)
         pod_spec = {**default_pod_spec, **pod_spec_override}
 
-        pod_spec['spec']['containers'][0]['image'] = ee.image
-        pod_spec['spec']['containers'][0]['args'] = ['ansible-runner', 'worker', '--private-data-dir=/runner']
```

Comment:
-1. See https://github.com/ansible/awx/issues/10568#issuecomment-872314993

Reply:
It's fine to put a default value if not set but I feel it's wrong when it's silently replaced.
User should be able to override to whatever value they'd like